IPNI.md

```diff
@@ -300,6 +307,17 @@ Specified protocols are expected to be ordered in increasing order.
 
 If the `Metadata` field is not specified, the advertisement is treated as address update only.
 
+#### SeqNum
```

Comment:
It seems like it'd be useful to return this in `/ad/head` as well, right (could also be by enabling returning just the root advertisement block either in the same or a separate call)?